### PR TITLE
Move tests outside ballerina module

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$title + Modify existing tests to use new stream types introduced (wrapper, timestamp etc)
## Examples
N/A
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [x] Added tests
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/1824

### CODE REVIEW

File: ballerina-tests/build.gradle [hunk 1]
```diff
@@ -0,0 +1,364 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (MadhukaHarith92) [inline ballerina-tests/build.gradle:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment (dilanSachi) [reply ballerina-tests/build.gradle]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/415/commits/e99db12aa0dc4da0c43e0d716cd183096359bf21

**Review by MadhukaHarith92 — APPROVED** [review]

File: ballerina-tests/tests/08_unary_service_with_headers.bal [hunk 2]
```diff
@@ -13,16 +13,17 @@
...
 // under the License.
-// This is server implementation for unary blocking/unblocking scenario
 
+import ballerina/grpc;
 import ballerina/io;
+import ballerina/protobuf.types.wrappers;
 
 // Server endpoint configuration
-listener Listener ep8 = new (9098, {
+listener grpc:Listener  ep8 = new (9098, {
```
Comment (BuddhiWathsala) [inline ballerina-tests/tests/08_unary_service_with_headers.bal:22]: Seems like an additional space has been added to all listeners.

File: ballerina-tests/tests/13_grpc_service_with_error_return.bal [hunk 3]
```diff
@@ -14,11 +14,12 @@
 // specific language governing permissions and limitations
 // under the License.
 
-@ServiceDescriptor {
+import ballerina/grpc;
```
Comment (BuddhiWathsala) [inline ballerina-tests/tests/13_grpc_service_with_error_return.bal:17]: Need space.

File: ballerina-tests/tests/40_bidirectional_streaming_negative_test_service.bal [hunk 4]
```diff
@@ -13,18 +13,19 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-import ballerina/log;
 
-listener Listener ep40 = new (9140);
+import ballerina/grpc;import ballerina/log;
```
Comment (BuddhiWathsala) [inline ballerina-tests/tests/40_bidirectional_streaming_negative_test_service.bal:17]: Need a new line.

File: ballerina-tests/tests/42_repeated_data_types_test_client.bal [hunk 5]
```diff
@@ -13,12 +13,13 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-import ballerina/test;
+
+import ballerina/grpc;import ballerina/test;
```
Comment (BuddhiWathsala) [inline ballerina-tests/tests/42_repeated_data_types_test_client.bal:17]: Need a new line

File: ballerina-tests/tests/08_unary_service_with_headers.bal (hunk 2 above)
Comment (dilanSachi) [reply ballerina-tests/tests/08_unary_service_with_headers.bal]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/415/commits/f82ce2a7dc39d85c31209f0c65ff33cd295e5441

File: ballerina-tests/tests/42_repeated_data_types_test_client.bal (hunk 5 above)
Comment (dilanSachi) [reply ballerina-tests/tests/42_repeated_data_types_test_client.bal]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/415/commits/27911584b1b415028a5454b5888d76ca4f315b2d

File: ballerina-tests/tests/40_bidirectional_streaming_negative_test_service.bal (hunk 4 above)
Comment (dilanSachi) [reply ballerina-tests/tests/40_bidirectional_streaming_negative_test_service.bal]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/415/commits/27911584b1b415028a5454b5888d76ca4f315b2d

File: ballerina-tests/tests/13_grpc_service_with_error_return.bal (hunk 3 above)
Comment (dilanSachi) [reply ballerina-tests/tests/13_grpc_service_with_error_return.bal:17]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/415/commits/27911584b1b415028a5454b5888d76ca4f315b2d

File: ballerina-tests/tests/test_commons.bal [hunk 6]
```diff
@@ -29,10 +29,10 @@ const string GENERATED_SOURCES_DIRECTORY = "build/generated-sources/";
...
 type BooleanTypedesc typedesc<boolean>;
 type FloatTypedesc typedesc<float>;
-type TestIntTypedesc typedesc<TestInt>;
-type TestStringTypedesc typedesc<TestString>;
-type TestBooleanTypedesc typedesc<TestBoolean>;
-type TestFloatTypedesc typedesc<TestFloat>;
-type TestStructTypedesc typedesc<TestStruct>;
-type ResponseTypedesc typedesc<Response>;
-type RequestTypedesc typedesc<Request>;
+//type TestIntTypedesc typedesc<TestInt>;
```
Comment (BuddhiWathsala) [inline ballerina-tests/tests/test_commons.bal:32]: Don't we need these types?

**Review by BuddhiWathsala — APPROVED** [review]

File: ballerina-tests/tests/test_commons.bal (hunk 6 above)
Comment (dilanSachi) [reply ballerina-tests/tests/test_commons.bal:32]: We need these types. I have missed to uncomment them. But the tests are compiled without an issue.

File: ballerina-tests/build.gradle [hunk 7]
```diff
@@ -0,0 +1,364 @@
...
+                    println("Gradle process can safely ignore stopGoServerDockerContainer task")
+                }
+            } else {
+                println "Golang gRPC server is not started."
+            }
+        }
+    }
+}
+
+publishing {
```
Comment (BuddhiWathsala) [inline ballerina-tests/build.gradle:342]: I'm not sure whether we need this publishing phase or not?

File: ballerina-tests/tests/test_commons.bal (hunk 6 above)
Comment (dilanSachi) [reply ballerina-tests/tests/test_commons.bal:32]: These types are already defined in https://github.com/dilanSachi/module-ballerina-grpc/blob/1d4ed52dd448e805b91769822911760fb31b328e/ballerina-tests/tests/resources/generated-sources/tool_test_data_type_20/multiple_wrapper_types_pb.bal#L553 apparently. This is why the tests compiled without an issue.

File: ballerina-tests/build.gradle (hunk 7 above)
Comment (dilanSachi) [reply ballerina-tests/build.gradle]: Removed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/415/commits/62b4a725c84f60a0f0070cae566c844235c28761

File: ballerina-tests/tests/test_commons.bal (hunk 6 above)
Comment (dilanSachi) [reply ballerina-tests/tests/test_commons.bal]: Removed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/415/commits/580e8664d8b02f8579c526273d308b94cb32bf73

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**